controller/src/main/java/io/pravega/controller/eventProcessor/impl/EventProcessorGroupImpl.java

```diff
@@ -199,40 +200,40 @@ protected void startUp() {
 
     @Override
     protected void shutDown() {
+        log.info("Shutting down all event processors in {}", this.toString());
         synchronized (lock) {
             long traceId = LoggerHelpers.traceEnterWithContext(log, this.objectId, "shutDown");
             try {
                 // If any of the following operations error out, the fact is just logged.
                 // Some other controller process is responsible for cleaning up the reader group,
                 // its readers and their position objects from checkpoint store.
                 try {
-                    log.info("Attempting to seal the reader group {} entry from checkpoint store", this.objectId);
+                    log.debug("Attempting to seal the reader group {} entry from checkpoint store", this.objectId);
                     checkpointStore.sealReaderGroup(actorSystem.getProcess(), readerGroup.getGroupName());
                 } catch (CheckpointStoreException e) {
                     log.warn("Error sealing reader group " + this.objectId, e);
                 }
 
                 // Initiate stop on all event processor cells and await their termination.
                 for (EventProcessorCell<T> cell : eventProcessorMap.values()) {
-                    log.info("Stopping event processor cell: {}", cell);
+                    log.debug("Terminating event processor cell: {}", cell);
```

Comment:
we can keep this at info level, it may be useful to know when a reader is shutdown in event processor

Reply:
ok. fixed